extras/data/machinery/generate/HERA.py

```diff
@@ -44,14 +44,19 @@ def dump_HERACOMB_heavy(src_path):
             target path
     """
     obs = obs_template.copy()
-    esf = load(src_path, 36, ["Q2", "x", "y"])
+    raw_esfs = load(src_path, 36, ["Q2", "x"])
+    esfs = []
+    for raw_esf in raw_esfs:
+        raw_esf.update({"y": raw_esf["Q2"] / 318.0**2 / raw_esf["x"]})
```

Comment:
Are you completely sure that $s = 318^2 GeV^2$?

Reply:
In case, please add a reference (if possible).

Reply:
For the two datasets at hand, yes - and I don't expect any new heavy HERA dataset to appear

Reply:
done in https://github.com/NNPDF/yadism/pull/196/commits/2cffb232be4327a57df5d4ac4a2774bd1d534c3d

Reply:
I was asking for those two datasets only. For future ones we'll take into account at future time.